src/layouts/Home.astro

```diff
@@ -39,6 +39,7 @@ const isVercelProd = import.meta.env.VERCEL_PROD;
 
     <link rel="icon" type="image/svg+xml" href="/favicon.png" />
 
+    <link type="text/plain" rel="author" href="http://domain/humans.txt" />
```

Comment:
Should `href="http://domain/humans.txt"` not be `/humans.txt`?

Reply:
oops.. That's also for the other layout too. I totally missed that. D: